src/disco/poh/fd_poh_tile.c

```diff
@@ -42,6 +42,7 @@ fd_poh_tile_publish_tick( fd_poh_tile_ctx_t * ctx,
   ulong tspub = (ulong)fd_frag_meta_ts_comp( fd_tickcount() );
   ulong sz = sizeof(fd_entry_batch_meta_t)+sizeof(fd_entry_batch_header_t);
   ulong sig = fd_disco_poh_sig( slot, POH_PKT_TYPE_MICROBLOCK, 0UL );
+  FD_LOG_WARNING(("PUBLISH TICK TO SHRED: %lu %lu", tick->hashcnt_delta, tick->txn_cnt ));
```

Comment:
Get rid of these warnings before merging, right?

Reply:
We are actively debugging using this branch right now, making it a draft